lib/exercism/tooling_job.rb

```diff
@@ -137,7 +164,7 @@ def s3_bucket_name
       Exercism.config.aws_tooling_jobs_bucket
     end
 
-    %w[queued locked executed processed cancelled].each do |key|
+    %w[queued locked executed cancelled].each do |key|
```

Comment:
The `processed` state is removed as Redis is now only a queue?

Reply:
Correct! :)